jaeger-core/src/main/java/io/jaegertracing/samplers/GuaranteedThroughputSampler.java

```diff
@@ -31,15 +30,15 @@
  */
 @ToString
 @EqualsAndHashCode
-@AllArgsConstructor (access = AccessLevel.PACKAGE) // Visible for testing
+@AllArgsConstructor
```

Comment:
Just noticed this change here: this exposes a *new* constructor, to initialize the private fields (`probabilisticSampler`, `lowerBoundSampler` and `tags`). 

I don't think we want that. In fact, I think this annotation can be removed altogether.